app.py

```diff
@@ -113,5 +113,5 @@
 
 
 if __name__ == '__main__':
-    app.run_server()
+    app.run_server(debug=True)
```

Comment:
Do we want to keep this? The debug mode on?

It will probably help contributors debug their contributions.

The drawback is that it will crowd the log from the github actions.